django/utils/connection.py

```diff
@@ -45,6 +45,10 @@ def settings(self):
         self._settings = self.configure_settings(self._settings)
         return self._settings
 
+    @property
+    def databases(self):
+        return self.settings
```

Comment:
Moving `databases` to the `BaseConnectionHandler` is misleading. I'd revert it.

Reply:
Oh yes sorry. When I readded this I apparently added it to the wrong place.